tests/unit/drivers/test_cache_content_driver.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pytest
+
+from jina import DocumentSet
+from jina.drivers.cache import BaseCacheDriver
+from jina.executors.indexers.cache import DocIDCache
+from jina.proto import jina_pb2
+from jina.types.document import uid, Document, bytes2id
+from tests import random_docs
+from tests.unit.drivers.test_cache_id_driver import MockCacheDriver
+
+
+def test_cache_content_driver(tmp_path):
```

Comment:
Yesterday, we had the example with for different documents, which collides on either `id` or `content`. Before continuing implementing, I'd put in the four use cases to have a complete view on what should be solved.

Reply:
Oh yeah, missed those. Will add them